src/Runtime/localtest/src/Controllers/HomeController.cs

```diff
@@ -588,7 +649,54 @@ private void CreateJwtCookieAndAppendToResponse(string identityCookie, int altin
                 HttpContext,
                 userSelectCookieBuilder.Name,
                 userSelect,
-                userSelectCookieOptions);
+                userSelectCookieOptions
+            );
+        }
+
+        private static bool IsDockerNetworkRange(IPAddress ipAddress)
+        {
+            if (ipAddress.AddressFamily != System.Net.Sockets.AddressFamily.InterNetwork)
+                return false;
+
+            var bytes = ipAddress.GetAddressBytes();
+
+            // 172.16.0.0/12 (172.16.0.0 - 172.31.255.255)
+            if (bytes[0] == 172 && bytes[1] >= 16 && bytes[1] <= 31)
+                return true;
+
+            // 192.168.0.0/16 (192.168.0.0 - 192.168.255.255)
+            if (bytes[0] == 192 && bytes[1] == 168)
+                return true;
+
+            return false;
+        }
+
+        private static IPAddress? GetDockerBridgeGateway()
+        {
+            try
+            {
+                var commonDockerGateways = new[]
+                {
+                    IPAddress.Parse("172.17.0.1"),
+                    IPAddress.Parse("172.18.0.1"),
+                    IPAddress.Parse("172.19.0.1"),
+                    IPAddress.Parse("172.20.0.1")
```

Comment:
This is a bit brittle, I think. It depends on how many docker networks you've set up, so if you have many this will break. Do we really need it, though? We kind-of require localtest to be running inside docker or podman at this point, and when doing so it only has _one_ default gateway. There should be no need to check if it's one of the common ones.